src/coffee/helpers/repeater.coffee

```diff
@@ -57,7 +57,9 @@ class Repeater
 
     debug '(re)-trying task, %d remaining attempts', remainingAttempts
     if remainingAttempts is 0
-      defer.reject new Error "Failed to retry the task after #{@_options.attempts} attempts. Cause: #{lastError.stack}"
+      defer.reject
```

Comment:
I need the whole `lastError` object. Only `lastError`'s stack is saved in the error object right now.